build-logic/build.gradle.kts

```diff
@@ -63,14 +64,9 @@ tasks.withType(KotlinJvmCompile::class.java).configureEach {
 
 gradlePlugin {
   plugins {
-    register("apollo.library") {
-      id = "apollo.library"
-      implementationClass = "com.apollographql.apollo3.buildlogic.plugin.LibraryConventionPlugin"
-    }
-
-    register("apollo.test") {
-      id = "apollo.test"
-      implementationClass = "com.apollographql.apollo3.buildlogic.plugin.TestConventionPlugin"
+    register("build.logic") {
+      id = "build.logic"
+      implementationClass = "build.logic.Plugin"
```

Comment:
The implementation class doesn't exist or am I missing something? (but how come applying the plugin works?)

Reply:
Sooooo.... I forgot to add the file to the commit. The fun stuff is that because the plugin is never applied, it still works. Gradle just needs the marker to resolve the .jar but the actual class is never loaded :). This is my new favorite Gradle hack! I'll add a comment.

Reply:
Hahaha nice one. Ooooh yeah there was `apply false` indeed!